lms/templates/help_modal.html

```diff
@@ -15,7 +15,7 @@
 % if settings.FEATURES.get('ENABLE_FEEDBACK_SUBMISSION', False):
 
 <div class="help-tab">
-  <a href="#help-modal" rel="leanModal" role="button">${_("Help")}</a>
+  <a href="#help-modal" rel="leanModal" role="button">${_("Feedback")}</a>
```

Comment:
Thanks for the review @sanfordstudent! I'm worried about making this change because that class name is sprinkled throughout five Sass files and in the JS for the feedback (née help) dialog. I opened https://openedx.atlassian.net/browse/DOC-3280 to clean up the class name. Would you support merging this PR with the name unchanged, and then doing the renaming work separately?